src/lib.rs

```diff
@@ -68,6 +68,7 @@ pub mod blockdata;
 pub mod util;
 pub mod consensus;
 pub mod hash_types;
+pub mod policy;
```

Comment:
Should we have this as a separate module, or it can just be a file inside `network`?

Reply:
I think this is fine, it's not really related to the p2p network layer imo, more general mempool policy.